csharp/.editorconfig

```diff
@@ -0,0 +1,233 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Guidelines ####
+guidelines = 120
+
+#### Core EditorConfig Options ####
```

Comment:
I assume these are the same options as in icerpc-csharp?

Reply:
Yes copied from icerpc-csharp repository.